bundle/libraries/libraries.go

```diff
@@ -44,18 +44,14 @@ func isEnvsWithLocalLibraries(envs []jobs.JobEnvironment) bool {
 	return false
 }
 
-func FindAllWheelTasksWithLocalLibraries(b *bundle.Bundle) []*jobs.Task {
+func FindTasksWithLocalLibraries(b *bundle.Bundle) []*jobs.Task {
 	tasks := findAllTasks(b)
 	envs := FindAllEnvironments(b)
 
 	wheelTasks := make([]*jobs.Task, 0)
 	for k, jobTasks := range tasks {
 		for i := range jobTasks {
 			task := &jobTasks[i]
-			if task.PythonWheelTask == nil {
-				continue
-			}
-
 			if isTaskWithLocalLibraries(*task) {
 				wheelTasks = append(wheelTasks, task)
 			}
```

Comment:
nit: The code below can be moved out of the loop
```
if envs[k] != nil && isEnvsWithLocalLibraries(envs[k]) {
  wheelTasks = append(wheelTasks, task)
}
```
Can instead be:
```
if envs[k] != nil && isEnvsWithLocalLibraries(envs[k]) {
  wheelTasks = append(wheelTasks, jobTasks...)
}
```

Reply:
Unfortunately we can't do that because `task` and `jobTasks...` are different types (1st one is reference, 2nd one is not)

Reply:
We don't seem to be modifying the pointer values `*jobs.Task` anywhere. Would be nicer to make FindAllWheelTasksWithLocalLibraries return a `[]jobs.Task`?

Reply:
You're right, updated the code